sloggers/slogjson/slogjson.go

```diff
@@ -2,19 +2,19 @@
 //
 // Format
 //
-//  {
-//    "ts": "2019-09-10T20:19:07.159852-05:00",
-//    "level": "INFO",
-//    "logger_names": ["comp", "subcomp"],
-//    "msg": "hi",
-//    "caller": "slog/examples_test.go:62",
-//    "func": "cdr.dev/slog/sloggers/slogtest_test.TestExampleTest",
-//    "trace": "<traceid>",
-//    "span": "<spanid>",
-//    "fields": {
-//      "my_field": "field value"
-//    }
-//  }
+//	{
```

Comment:
this gofmt change has been driving us crazy at @terrastruct

We refuse to use go 1.19 for now because of it. Ruined all our ascii diagrams lol

Reply:
Lol, I was quite confused by this.